.env.example

```diff
@@ -19,4 +19,7 @@ POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
-POSTGRES_DB=postgres
\ No newline at end of file
+POSTGRES_DB=postgres
+
+# Status Page
+STATUSPAGE_API_KEY=""
```

Comment:
What this status page api key means, just curious, maybe I don't understand the variable and it may be renamed, or just the mean of it?

Reply:
Due to secured /users and /ping endpoints, the status page needs an authorized access to them. So STATUSPAGE_API_KEY can be used to give him the authorized access.